packages/react-make-styles/tsconfig.json

```
@@ -1,22 +1,17 @@
 {
+  "extends": "../../tsconfig.base.json",
+  "include": ["src"],
   "compilerOptions": {
-    "baseUrl": ".",
+    "target": "ES5",
+    "module": "CommonJS",
+    "lib": ["es5", "dom"],
```

`build:local` will throw error. this is issue @ling1726  run into already (https://github.com/microsoft/fluentui/issues/18682). 

**context:** make-styles is using `ES2015` language features, thus this package needs to use the same

```suggestion
    "lib": ["ES2015", "dom"],
```